datasets/energielabel/dataset.json

```diff
@@ -0,0 +1,191 @@
+{
+  "id": "energielabel",
+  "type": "dataset",
```

Comment:
Je kunt de afscherming zelf regelen met een `"auth"` scope. Voor medewerker gemeente Amsterdam is dat `"FP/MDW"` (FP = functieprofiel). Zie bijv.: https://schemas.data.amsterdam.nl/datasets/bor_inspecties/dataset

Reply:
Dank je, ik heb het aangepast.